radio/src/translations/ua.h

```diff
@@ -0,0 +1,1257 @@
+#define TR_VOICE_ITALIANO               "Італійська"
+#define TR_VOICE_POLISH                 "Польська"
+#define TR_VOICE_PORTUGUES              "Португальська"
+#define TR_VOICE_RUSSIAN                "Pосійський"
+#define TR_VOICE_SLOVAK                 "Словацька"
+#define TR_VOICE_SWEDISH                "Шведська"
+#define TR_VOICE_TAIWANESE              "Тайська"
+#define TR_VOICE_JAPANESE               "Японська"
+#define TR_VOICE_HEBREW                 "Іврит"
+#define TR_VOICE_UKRAINIAN              "Українська"
+
+#define TR_USBJOYSTICK_LABEL           "USB Джойстик"
+#define TR_USBJOYSTICK_EXTMODE         "Режим"
+#define TR_VUSBJOYSTICK_EXTMODE        "Класичний","Просунутий"
+#define TR_USBJOYSTICK_SETTINGS        "Установки каналу"
+#define TR_USBJOYSTICK_IF_MODE         TR("Режим ІФ","Режим інтерфейсу")
+#define TR_VUSBJOYSTICK_IF_MODE        "Джойстик","Gamepad","MultiAxis"
+#define TR_USBJOYSTICK_CH_MODE         "Режим"
+#define TR_VUSBJOYSTICK_CH_MODE        "Немає","Btn","Axis","Sim"
+#define TR_VUSBJOYSTICK_CH_MODE_S      "-","B","A","S"
+#define TR_USBJOYSTICK_CH_BTNMODE      "Режим кнопок"
+#define TR_VUSBJOYSTICK_CH_BTNMODE     "Нормальн","Імпульс","SWEmu","Дельта","Компаньон"
+#define TR_VUSBJOYSTICK_CH_BTNMODE_S   TR("Норм","Нормальний"),TR("Імп","Імпульс"),TR("SWEm","SWEmul"),TR("Дельта","Дельта"),TR("КПН","Компаньйон")
+#define TR_USBJOYSTICK_CH_SWPOS        "Позиції"
+#define TR_VUSBJOYSTICK_CH_SWPOS       "Push","2POS","3POS","4POS","5POS","6POS","7POS","8POS"
+#define TR_USBJOYSTICK_CH_AXIS         "Вісь"
+#define TR_VUSBJOYSTICK_CH_AXIS        "X","Y","Z","rotX","rotY","rotZ","Slider","Dial","Wheel"
+#define TR_USBJOYSTICK_CH_SIM          "Сим. вісі"
+#define TR_VUSBJOYSTICK_CH_SIM         "Ail","Ele","Rud","Thr"
+#define TR_USBJOYSTICK_CH_INVERSION    "Інверсія"
+#define TR_USBJOYSTICK_CH_BTNNUM       "Номер Кнопки"
+#define TR_USBJOYSTICK_BTN_COLLISION   "!Конфлікт номерів кнопок!"
+#define TR_USBJOYSTICK_AXIS_COLLISION  "!Конфлікт осей!"
+#define TR_USBJOYSTICK_CIRC_COUTOUT    TR("Круговий виріз", "Круговий виріз")
+#define TR_VUSBJOYSTICK_CIRC_COUTOUT   "Нема","X-Y, Z-rX","X-Y, rX-rY"
+#define TR_USBJOYSTICK_APPLY_CHANGES   "Застосувати зміни"
+
+#define TR_DIGITAL_SERVO          "Серво 333HZ"
+#define TR_ANALOG_SERVO           "Серво 50HZ"
+#define TR_SIGNAL_OUTPUT          "Вихід сигналу"
+#define TR_SERIAL_BUS             "Серійна шина"
+#define TR_SYNC                   "Сихро"
+
+#define TR_ENABLED_FEATURES       "Активовані функції"
+#define TR_RADIO_MENU_TABS        "Вкладки меню радіо"
+#define TR_MODEL_MENU_TABS        "Вкладки меню моделей"
+
+#define TR_SELECT_MENU_ALL        "Все"
+#define TR_SELECT_MENU_CLR        "Очистити"
+#define TR_SELECT_MENU_INV        "Інвертувати"
+
+#define TR_SORT_ORDERS            "Name A-Z","Name Z-A","Least used","Most used"
+#define TR_SORT_MODELS_BY         "Sort models by"
+#define TR_CREATE_NEW             "Create"
```

Comment:
Translated these phrases:
```suggestion
#define TR_SORT_ORDERS            "Назва A-Z","Назва Z-A","Найменш використ.","Найбільш використ."
#define TR_SORT_MODELS_BY         "Сортувати моделі за"
#define TR_CREATE_NEW             "Створити"
```